tests/unit/src/main/scala/tests/TestRanges.scala

```diff
@@ -27,4 +30,23 @@ object TestRanges extends RangeReplace {
       }
     resolved.toMap
   }
+
+  def renderEditAsString(
+      file: String,
+      code: String,
+      workspaceEdit: WorkspaceEdit
+  ): Option[String] = {
+    for {
+      validLocations <- workspaceEdit
+        .getDocumentChanges()
+        .asScala
+        .find(
+          change =>
+            change.isLeft && change.getLeft.getTextDocument.getUri
```

Comment:
nit: 
```suggestion
            change.isLeft &&
            change.getLeft.getTextDocument.getUri.contains(file)
```

Scalafmt doesn't insert newlines in infix operators

Reply:
Fixed